.docs/user-guide/automation.md

```diff
@@ -14,7 +14,32 @@ orchestration tools below.  Optionally, Docker is also listed in some examples.
 
 <br>
 ## Ansible
-ToDo
+With Ansible, installing the latest REX-Ray binaries can be accomplished by
+including the `codenrhoden.rexray` role from Ansible Galaxy.  The role accepts
+all the necessary variables to properly fill out your `config.yml` file.
```

Comment:
No problem!  Thanks for the merge.

I'll update the other piece to add the `$` and change the path to the role in a second PR once the repo move has happened.